packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md

```diff
@@ -196,7 +196,17 @@ Lists all Custom Authorization Servers in this Okta organization
 
 ##### Request parameters
 
-None
+
+Parameter | Description                                                                                | Param Type | DataType | Required | Default
+--------- | ------------------------------------------------------------------------------------------ | ---------- | -------- | -------- | -------
+q         | Searches the `name` and `audiences` of Authorization Servers for matching value            | Query      | String   | FALSE    |
+limit     | Specifies the number of Authorizaton Server results in a page                              | Query      | Number   | FALSE    | 200
+after     | Specifies the pagination cursor for the next page of Authorization Servers                 | Query      | String   | FALSE    |
+
+Parameter Details
+
+* The `after` cursor should treated as an opaque value and obtained through the next link relationship. See [Pagination](/docs/reference/api-overview/#pagination).
```

Comment:
adding "be"

The `after` cursor should be treated as an opaque value and obtained through the next link relationship.